@xen-orchestra/web/src/pages/vm/new.vue

```diff
@@ -781,16 +779,16 @@ const vmData = computed(() => {
     name_description: vmState.description,
     name_label: vmState.name,
     template: vmState.new_vm_template?.uuid,
-    vdis: vmState.vdis.map(disk => ({
-      ...disk,
-      size: giBToBytes(disk.size),
+    vdis: [...vmState.vdis, ...vmState.existingVdis].map(vdi => ({
+      ...vdi,
+      size: giBToBytes(vdi.size),
```

Comment:
You should don't send all properties for `existingVdis` but only props that need to be updated (except for the userdevice).
The REST API create all `existingVdis` himself, then edit them if needed. So if you send an `existingVdis` to the REST API with `size`, `name_label`, `name_description`, it will try to update them. (Event if the user didn't change it.)

ideally you should also only send VDIs that need to be updated (instead of sending all existing VDIs)